src/region-bitmap.c

```diff
@@ -537,9 +537,7 @@ gdip_region_bitmap_from_path (GpPath *path)
 
 	/* here the memory is allocated for a ARGB bitmap - so 32 times bigger than our normal alpha bitmap */
 	if ((size < 1) || (size > REGION_MAX_BITMAP_SIZE << 5)) {
-		g_warning ("Path conversion requested %lu bytes (%d x %d). Maximum size is %d bytes.",
-			size, bounds.Width, bounds.Height, REGION_MAX_BITMAP_SIZE << 5);
-		return NULL;
+		return alloc_bitmap_with_buffer (REGION_INFINITE_POSITION, REGION_INFINITE_POSITION, REGION_INFINITE_LENGTH, REGION_INFINITE_LENGTH, NULL);
```

Comment:
Not sure about this. How is it handled by `alloc_merged_bitmap` and all the region combine operations? Are we sure it is never reached?